config/checkstyle_checks.xml

```diff
@@ -55,6 +55,9 @@
     <property name="offCommentFormat" value="CSOFF\: .*"/>
     <property name="onCommentFormat" value="CSON\: .*"/>
   </module>
+  <module name="SuppressWithNearbyPlainTextCommentFilter">
+    <!-- TO BE DETERMINED IN PULL REQUEST -->
+  </module>
```

Comment:
This needs to be discussed.

Reply:
original issue is https://github.com/checkstyle/checkstyle/issues/7064 and it mentioned problem with LineLength 
so we can find suppression on Linelenght and use this filter instead.

Reply:
Absolutely! Added a commit that does just that.